doc/src/modules/assumptions/index.rst

```diff
@@ -32,7 +32,7 @@ assumptions about objects in expr. Valid values include:
     * Q.integer(x) & Q.positive(x)
     * etc.
 
-Q is a class in sympy.assumptions holding known predicates.
+Q is an instance of QClass in sympy.assumptions.ask holding known predicates.
```

Comment:
I agree the class should not be part of the API, and like any class with the word "Class" in its name, it probably could use a better name :)